app/js/daemonManager.js

```diff
@@ -72,17 +72,21 @@ function DaemonManager() {
 		// duration that it takes to load up the blockchain.
 		apiCall("/daemon/updates/check", function(err, update) {
 			if (err) {
+				console.log(err, update)
 				self.Running = false;
 				// Check again later
-				setTimeout(updatePrompt, 1000);
+				setTimeout(updatePrompt, 5000);
```

Comment:
If you look at pluginManager.js, I implemented a trick where if an apiCall fails, it will trigger Daemon.ifSiad() which will flip Daemon.Running based on the result.

You could add another ipc channel specifically for this, though